src/main.js

```diff
@@ -51,14 +53,13 @@ function showNewRandomCover() {
 function createNewCover() {
   makeNewFormLocation.classList.remove('hidden');
   homePageLocation.classList.add('hidden');
-  viewSavedCoverLocation.classList.add('hidden');
+  viewSavedViewLocation.classList.add('hidden');
   randomCoverButton.classList.add('hidden');
   saveCoverButton.classList.add('hidden');
   homeButton.classList.remove('hidden');
 }
 function viewSavedCovers() {
-  viewSavedCoverLocation.classList.remove('hidden');
-  console.log(savedCovers);
+  viewSavedViewLocation.classList.remove('hidden');
```

Comment:
Appears there is a lot of repetition around hiding and displaying elements - consider whether it makes sense to refactor and dry this up.